software/plexripper

```diff
@@ -0,0 +1,11 @@
+name: "PlexRipper"
+website_url: "https://www.plexripper.rocks/"
+source_code_url: "https://github.com/PlexRipper/PlexRipper"
+description: "A cross-platform Plex media downloader that seamlessly adds media from other Plex servers to your own!"
```

Comment:
```suggestion
description: "A cross-platform Plex media downloader that seamlessly adds media from other Plex servers to your own."
```

Oops, forgot another thing. Descriptions need to end with a dot, otherwise, our checks will fail. Sorry..

Reply:
Sure, no problem!.